packages/clerk-js/src/core/warnings.ts

```diff
@@ -0,0 +1,23 @@
+const formatWarning = (msg: string) => {
```

Comment:
Grouped all warnings into a single file so we can resuse the errorThrower util in a follow up PR more easily

Reply:
Another benefit is that grouping these long messages will make it easier for us to verify that webpack correctly prune them for prod builds, once the infra is in place